dbms/src/Storages/KVStore/MultiRaft/Disagg/FastAddPeer.h

```diff
@@ -62,6 +95,10 @@ class FastAddPeerContext
     // StoreId -> std::pair<UploadSeq, CheckpointCacheElementPtr>
     std::unordered_map<UInt64, std::pair<UInt64, CheckpointCacheElementPtr>> checkpoint_cache_map;
 
+    // Checkpoint that is persisted, but yet to be ingested into DeltaTree.
+    std::mutex ingest_info_mu;
+    std::map<UInt64, CheckpointIngestInfoPtr> checkpoint_ingest_info_map;
```

Comment:
Why use `std::map` instead of `std::unordered_map`?

Reply:
fixed